src/style.css

```diff
@@ -1,3 +1,93 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
+
+:root {
+  --accent-color: #00acdc;
+  --bg-color: white;
+  --text-color: black;
+  --roboto-font: 'Roboto', sans-serif;
+  --header-size: 60px;
+}
+
+html {
+  font-family: var(--roboto-font);
+  padding: 0;
+  margin: 0;
+}
+
 body {
+  padding: 0;
+  margin: 0;
+}
+
+ul {
+  padding: 0;
+  margin: 0;
+}
+
+li {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). For list items, use a class like `.nav__item` to apply the styles.